src/GitVersion.Core/Git/ReferenceName.cs

```diff
@@ -13,6 +13,8 @@ public class ReferenceName : IEquatable<ReferenceName?>, IComparable<ReferenceNa
     private const string TagPrefix = "refs/tags/";
     private const string PullRequestPrefix1 = "refs/pull/";
     private const string PullRequestPrefix2 = "refs/pull-requests/";
+    private const string RemotePullRequestPrefix1 = "refs/remotes/pull/";
+    private const string RemotePullRequestPrefix2 = "refs/remotes/pull-requests/";
```

Comment:
If these prefixes are usually checked together, perhaps we can group them into a single array instead?

Reply:
Moved to `PullRequestPrefixes` array and also added a method `IsPrefixedBy`.